kong/plugins/influxdb-log/handler.lua

```diff
@@ -0,0 +1,105 @@
+local HttpLogHandler = require "kong.plugins.http-log.handler"
+
+local InfluxdbLogHandler = HttpLogHandler:extend()
```

Comment:
I guess my point is that I don't really like inter-plugins dependencies, what do you think @thibaultcha ?